server/lib/get_user_database.js

```diff
@@ -0,0 +1,23 @@
+const pgClient = require('./pg_client');
+
+
+const getUserDatabase = (connection, userId, callback) => {
+  const client = pgClient(connection);
+  client.query(
+      `SELECT * FROM users
+       WHERE id = $1`,
+      [userId],
+    (postgresErr, result) => {
+      if (postgresErr) {
+        callback(postgresErr);
+      } else {
+        callback(postgresErr, result);
+      }
+      client.end();
```

Comment:
If we use pg.Pool we don't need to end the client, and can manage a central pool of connections (so stop the app from crashing if too many connections opened).